checker/src/org/checkerframework/checker/lowerbound/LowerBoundAnalysis.java

```diff
@@ -0,0 +1,45 @@
+package org.checkerframework.checker.lowerbound;
+
+import java.util.List;
+import javax.lang.model.element.VariableElement;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.flow.CFAbstractAnalysis;
+import org.checkerframework.framework.flow.CFStore;
+import org.checkerframework.framework.flow.CFValue;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.javacutil.Pair;
+
+/**
+ * This class doesn't really have anything unique or interesting
```

Comment:
Ok, so you either need this class here or you need to create an anonymous class in LowerBoundAnnotatedTypeFactory#createFlowAnalysis.  So, you could keep this class if you prefer.